packages/obonode/obojobo-chunks-question-bank/import-questions-modal.scss

```diff
@@ -0,0 +1,57 @@
+@import '~styles/includes';
+
+.import-questions-model {
+	max-width: 70vw;
+
+	.import-model--question-content {
+		background: $color-bg2;
+		border-radius: $dimension-rounded-radius;
+		max-height: 55vh;
+		min-width: 35em;
+		overflow-y: scroll;
```

Comment:
Super minor, but I think this would look a little cleaner with `overflow: scroll` so we don't have an empty scrollbar when the dialog doesn't have a lot of content.